dashboard/test/ui/step_definitions/blockly.rb

```diff
@@ -47,17 +47,49 @@
 
 Then /^block "([^"]*)" is at offset "([^"]*), ([^"]*)"$/ do |block, x, y|
   point = get_block_coordinates(get_block_id(block))
-  x.to_i.should eq point.x
-  y.to_i.should eq point.y
+  point.x.should eq x.to_i
+  point.y.should eq y.to_i
```

Comment:
This test failed, and produced confusing output, so I reversed it.  Now if you write
```
Then block "1" is at offset "50, 70"
```
and it fails, it will say `Expected 50, got 30` where before it said `Expected 30, got 50`.